test/e2e/data/infrastructure-aws/withoutclusterclass/e2e_test_templates/cluster-template-internal-elb.yaml

```diff
@@ -28,6 +28,12 @@ spec:
   controlPlaneLoadBalancer:
     scheme: internal
   network:
+    cni:
+      cniIngressRules:
```

Comment:
IIRC there is a default set of rules that are used when this list is empty, that is not used at all when the list has a single item, you may need to include those default rules to allow the default CNI to continue to function